README.md

```diff
@@ -72,6 +74,27 @@ popolo.organizations.where(classification: "party")
     #    ]
 ```
 
+### See also: everypolitician-ruby
+
+In the example above, the Popolo data comes from a downloaded file
+(`ep-popolo-v1.0.json`), which is the kind of file you can get from the 
+[EveryPolitician website](http://everypolitician.org).
+But your Ruby application can also interact directly with the EveryPolitician
+data using the
+[everypolitician-ruby gem](http://github.com/everypolitician/everypolitcian-ruby),
+so you don't need to handle JSON files at all. The data the gem returns is in
+`Everypolitician::Popolo` format.
+
+```ruby
+require 'everypolitician'
+
+australia = Everypolitician.country('Australia')
+australia.code # => "AU"
+senate = australia.legislature('Senate')
+senate.persons.find_by(name: "Aden Ridgeway", openaustralia: "10540")
```

Comment:
I realise we have the same problem elsewhere in this file (which we should also fix), but I don't think we should show this style of searching by multiple fields in one shot as the primary example. If we want to show that you can search by more things than name, I'd suggest giving two examples, rather than one.